sys-freebsd/boot0/files/boot0-add-nossp-cflags.patch

```diff
@@ -0,0 +1,11 @@
+https://bugs.gentoo.org/show_bug.cgi?id=511698
```

Comment:
I'll merge it as-is because of the urgency but next time (or maybe fix it later), please put some textual explanation. Pure links aren't good since they require accessing external resource.